### DISCUSSION TITLE
Overload for no base styles

### DISCUSSION THREAD

**Discussion by alii** [joe-bell/cva#20] [Ideas]
**Is your feature request related to a problem? Please describe.**
Frustrating/pointless to pass a blank string to CVA as the first argument when it's not used

**Describe the solution you'd like**
To be able to use CVA to maintain consistency across the app for styling but with zero base styles

**Describe alternatives you've considered**
n/a

**Additional context**
Example:
```ts
const navlink = cva({
  variants: {},
});
```

**Comment by joe-bell** [comment]
I’m inclined to agree, but I would consider this fairly non-urgent, and I don’t have the bandwidth to add this right now. Let’s move this to a discussion and see if others feel the same

**Reply by alii** [reply]
Yep no problem! Thank you for making cva :D

I could have a stab at a PR if that could be useful?
